doc/api/readline.md

```diff
@@ -195,6 +195,9 @@ The `rl.close()` method closes the `readline.Interface` instance and
 relinquishes control over the `input` and `output` streams. When called,
 the `'close'` event will be emitted.
 
+Calling `rl.close()` does not immediately stop other events (including `'line'`)
```

Comment:
Micro-nit: Is `immediately` meaningful here? Would it be better without it?

Reply:
Instead of `immediately`, might `necessarily` be better?

Reply:
I copied the text from the `pause()` docs for consistency. That said, I'm open to using any wording here if you feel strongly enough.

Reply:
It’s a nit. I don’t feel strongly.